package.json

```diff
@@ -24,7 +24,7 @@
   "scripts": {
     "preinstall": "./scripts/download-mapbox-gl-native-ios.sh 3.2.0",
```

Comment:
Ok. Then I'll check it out when I get home.
On Fri, 22 Jul 2016 at 11:08 Bobby Sudekum <notifications@github.com> wrote:

> In package.json
> <https://github.com/mapbox/react-native-mapbox-gl/pull/365#discussion_r71841713>
> :
>
> > @@ -24,7 +24,7 @@
> >    "scripts": {
> >      "preinstall": "./scripts/download-mapbox-gl-native-ios.sh 3.2.0",
>
> @dapetcu21 <https://github.com/dapetcu21> I'd like to get both android
> and ios on the latest version. We just released a big bug fix release
> <http://status.mapbox.com/incidents/59f09r86vhyt> and I'd like to make
> sure we're on this. Theoretically, there should be no breaking changes
> since this branch is already on 3.x.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/mapbox/react-native-mapbox-gl/pull/365/files/d288f274da1f528f6d4fce856b999ecadc240bab#r71841713>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAaIHFvr0fln1DMn2uhGxa34B8qlMXrnks5qYHqQgaJpZM4I4ROX>
> .
>